General queries/Baseline Comparison.txt

```diff
@@ -0,0 +1,239 @@
+// Baseline Comparison
+// Author: miflower
+// The purpose of this query is to perform a comparison between "known good" machines and suspected bad machines
+// The original concept for this query was born due to reapplying the same 'whitelist' filters over and over
+// It brings deltas between a baseline and another machine quickly to the analyst's view
+// This query supports multiple suspected bad machines and multiple "known good" machines
+// It also supports providing a timeframe for how far back in time to build a baseline as well as how far back in time to evaluate the suspected bad machines
+// Each of the links provided by machineid/computername will go to the most recent entry for whatever entity is listed
+// Average results for the pre-defined settings below with a single good host and a single bad host:
```

Comment:
These stats are DSRE specific (especially the compute time).
Either remove it or comment that it's the compute time on a huge tenant.

Reply:
Added comment ;)